docs/UserGuide.md

```diff
@@ -168,10 +168,11 @@ Examples:
 
 Adds a task for a datetime with a tag.
 
-Format: `addt tn/TASKNAME dt/TIME t/TAG`
+Format: `addt tn/TASKNAME dt/DATETIME[, ENDDATETIME] t/TAG`
```

Comment:
Maybe change dt/DATETIME[, ENDDATETIME] to dt/DATETIME, ENDDATETIME

Reply:
Enddatetime is optional tho

Reply:
I see 👌 Overall LGTM, can merge it